core/src/main/java/com/microsoft/applicationinsights/internal/quickpulse/DefaultQuickPulseDataFetcher.java

```diff
@@ -50,13 +50,13 @@ public DefaultQuickPulseDataFetcher(ArrayBlockingQueue<HttpRequest> sendQueue, T
         sdkVersion = getCurrentSdkVersion();
         StringBuilder sb = new StringBuilder();
 
-        // FIXME (trask) what about azure functions consumption plan where role name not available yet?
+        // FIXME (heya) what about azure functions consumption plan where role name not available yet?
         String roleName = telemetryClient.getRoleName();
 
         sb.append("[{");
         sb.append("\"Documents\":[],");
         sb.append("\"Instance\":\"").append(instanceName).append("\",");
-        // FIXME (trask) this seemed to be working when it was always null ikey here??
+        // TODO (trask) instrumentation key is also part of query string, is it needed in both places?
```

Comment:
it always sends null instrumentationKey in the live metric payload. 
ikey is only used in the uri.